packages/iov-bns-governance/src/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { CommitteeId } from "./proposals";
+
+export interface CommitteeIds {
+  readonly GOVERNING_BOARD: CommitteeId;
```

Comment:
Can we use `lowerCamelCase` for consistently all values and properties? So far, we could avoid using SCREAM_CASE, which is a good thing.